operator/hack/deploy-gcp-storage-secret.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eo pipefail
```

Comment:
I think I would prefer having `nounset` in the options as well, but this requires changes to the error message logic below:

```suggestion
set -euo pipefail
```

--- 

side-note: I noticed the other scripts have `set -eou pipefail` in them ... I'm actually surprised that this works, as `pipefail` would now be an option for `u` and not `o`. Can you fix this in a drive-by?